src/utils/time.ts

```diff
@@ -256,3 +256,18 @@ export function parseTimeString(value: string): Time {
     second: null
   };
 }
+
+export function toISOTimeString(value: string, includeSeconds = true): string {
+  if (!isValidTime(value)) {
+    return "";
+  }
+  const { hour, minute, second } = parseTimeString(value);
+
+  let result = `${formatTimePart(parseInt(hour))}:${formatTimePart(parseInt(minute))}`;
```

Comment:
Nitpick: is there a more expressive name we could use instead of `result`?

Reply:
`returnValue`?

Reply:
I like `isoTimeString` better, actually.